packages/core/test/lib/commands/obtain.js

```diff
@@ -18,22 +17,29 @@ describe("obtain", () => {
       CompilerSupplier.prototype.downloadAndCacheSolc.restore();
     });
 
-    it("calls downloadAndCacheSolc on the supplier with the version", async () => {
-      await command.run(options, done);
+    it("calls downloadAndCacheSolc on the supplier with the version", async function () {
+      await command.run(options);
       assert(
         CompilerSupplier.prototype.downloadAndCacheSolc.calledWith("0.5.3")
       );
     });
 
-    describe("when options.solc is present", () => {
+    describe("when options.solc is not present", async () {
```

Comment:
Typo here (was this my fault? oops)

```suggestion
    describe("when options.solc is not present", async function () {
```